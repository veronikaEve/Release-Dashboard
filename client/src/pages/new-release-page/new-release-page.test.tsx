import { render } from "@testing-library/react";

import NewReleasePage from "./new-release-page";

describe("NewReleasePage", () => {
  test("should match snapshot", () => {
    const { asFragment } = render(<NewReleasePage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
