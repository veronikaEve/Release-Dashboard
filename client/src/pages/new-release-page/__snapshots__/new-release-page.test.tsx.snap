// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewReleasePage should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="col-md-4"
  >
    <div
      class="release-info-card text-center card"
    >
      <div
        class="card-header"
      >
        Release Info
      </div>
      <div
        class="card-body"
      >
        <div
          class="info-group row"
        >
          <div
            class="info-group__label col"
          >
            Branch name
          </div>
          <div
            class="info-group__info col"
          >
            <select
              as="select"
              class="form-select"
            >
              <option>
                Choose a branch
              </option>
            </select>
          </div>
        </div>
        <div
          class="info-group row"
        >
          <div
            class="info-group__label col"
          >
            Current version
          </div>
          <div
            class="info-group__info col"
          >
            1.4
          </div>
        </div>
        <div
          class="info-group row"
        >
          <div
            class="info-group__label col"
          >
            Commit hash
          </div>
          <div
            class="info-group__info col"
          />
        </div>
        <div
          class="info-group row"
        >
          <div
            class="info-group__label col"
          >
            Last Update
          </div>
          <div
            class="info-group__info col"
          />
        </div>
        <div
          class="info-group row"
        >
          <div
            class="info-group__label col"
          >
            Opened By
          </div>
          <div
            class="info-group__info col"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
